### PR TITLE
Example to parse values from JSON response to HTTP Request

### PR DESCRIPTION
**Pull request by nagyrobi**
## Description:

**Related issue (if applicable):** fixes <link to issue>

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#<esphome PR number goes here>

## Checklist:

  - [ ] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/http_request.rst [hunk 1]
```diff
@@ -14,6 +14,7 @@ The ``http_request`` component lets you make HTTP/HTTPS requests. First, you nee
     # Example configuration entry
     http_request:
       useragent: esphome/device
+      id: http_request_data
```
Comment (jesserockz) [inline components/http_request.rst]: I don't like setting the `id` like this on the main example, especially when it is only used way down below in the new example...

Comment (nagyrobi) [reply components/http_request.rst]: Removed

File: components/http_request.rst [hunk 2]
```diff
@@ -217,6 +217,29 @@ as seen below.
         # Will send:
         # {"key": 42.0, "greeting": "Hello World"}
 
+GET values from a JSON body response
+************************************
+
+The server returns a response in a JSON object over HTTP like this:
+``{"type":"0","ch":"0","mode":"40","loop":"0","eq":"0","status":"play","curpos":"7133","offset_pts":"7133","totlen":"0","Title":"556E6B6E6F776E","Artist":"556E6B6E6F776E","Album":"556E6B6E6F776E","alarmflag":"0","plicount":"0","plicurr":"0","vol":"42","mute":"0"}``
```
Comment (flaviut) [inline components/http_request.rst]: Can you please simplify this example to match those above? esphome.io domain, a very short data object, etc?

Comment (nagyrobi) [reply components/http_request.rst]: I thought a real world working example is always better.

Comment (jesserockz) [reply components/http_request.rst]: While that can be true, a short, concise, easy to understand example is always better for users.

Comment (nagyrobi) [reply components/http_request.rst]: Done.

**Comment by flaviut** [comment]
Thank you :)

**Pull request merged**